lib/circles/create.js

```diff
@@ -3,10 +3,10 @@ const fullStateEnum = require("./fullStateEnum");
 module.exports = function(req, res) {
     const circle = {
         name: req.body.name || null,
-        vision: req.body.vision || null,
-        mission: req.body.mission || null,
-        aim: req.body.aim || null,
-        expectationsForMembers: [],
+        vision: req.body.vision || "",
+        mission: req.body.mission || "",
+        aim: req.body.aim || "",
+        expectationsForMembers: req.body.expectationsForMembers || "",
```

Comment:
If we explicitly unset them in db (I think we have to set them as `undefined` here, then) we should make sure to set them to empty strings on calls to GET /circles/{id} and GET /circles